src/Features/VisualBasic/Portable/RemoveUnusedMembers/VisualBasicRemoveUnusedMembersDiagnosticAnalyzer.vb

```diff
@@ -9,5 +9,36 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.RemoveUnusedMembers
     <DiagnosticAnalyzer(LanguageNames.VisualBasic)>
     Friend NotInheritable Class VisualBasicRemoveUnusedMembersDiagnosticAnalyzer
         Inherits AbstractRemoveUnusedMembersDiagnosticAnalyzer(Of DocumentationCommentTriviaSyntax, IdentifierNameSyntax)
+
+        Protected Overrides Sub RegisterCustomSymbolReferenceActions(context As SymbolStartAnalysisContext, onSymbolUsageFound As Action(Of ISymbol, ValueUsageInfo))
+            ' Mark all methods with handles clause as having a read reference
+            ' to ensure that we consider the method as "used".
+            For Each method In DirectCast(context.Symbol, INamedTypeSymbol).GetMembers().OfType(Of IMethodSymbol)
```

Comment:
💭 It's not clear from the enclosing method signature that this is a valid cast. I couldn't think of an alternative name or signature which is "obviously better".

:bulb: Actually, `HandleNamedTypeSymbolStart` seems like a good name.